### DISCUSSION TITLE
如何手动启用视频字幕功能？

### DISCUSSION THREAD

**Discussion by Ptilopsis01** [immersive-translate/immersive-translate#1409] [Help]
### 概述 | Summary

如题，设置页面有“手动点击开启字幕”的字样
<img width="527" alt="image" src="https://github.com/immersive-translate/immersive-translate/assets/93318731/c3a4cc2e-fae6-4bb6-b86f-a316a0c6566f">
<img width="553" alt="image" src="https://github.com/immersive-translate/immersive-translate/assets/93318731/beecfa85-3480-4991-acee-a7264c639976">

但实际上在播放器中只有“自动启用”和“不自动启用”选项，并没有可手动开启字幕的操作

**Comment by theowenyoung** [comment]
目前的实现逻辑是劫持原视频的字幕流，所以不太好做到手动启用。

未来可能会更改我们的实现方案，改为使用我们自定义的字幕条，那种情况下就可以实现手动启用了。

**Reply by Ptilopsis01** [reply]
其实我的意思是需要一个开关可即时控制当前视频是否翻译，现在这个情况我开翻译就会导致之后看视频默认全都翻译，关翻译之后就全部默认不翻译，这就很麻烦
